static/js/match_play.js

```diff
@@ -336,6 +324,5 @@ $(function() {
     eventStatus: function(event) { handleEventStatus(event.data); },
     matchTime: function(event) { handleMatchTime(event.data); },
     matchTiming: function(event) { handleMatchTiming(event.data); },
-    realtimeScore: function(event) { handleRealtimeScore(event.data); },
```

Comment:
is it still useful to have this page auto-update when scores change during a match?

Reply:
Maybe comment this as a fixme instead of deleting, then?